Home-work/03-task.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Task 3</title>
+</head>
+
+<body>
+    <pre>
+        Задание:
+        Доработайте сценарий. Перед выводом общей стоимости заказа узнайте у пользователя, будет ли он заказывать доставку. 
+        Если да, добавьте к стоимости заказа 40.
+    </pre>
+    <script>
+        let product1 = "Бумага офисная А4, 80 г/м2, 500 л";
+        let product2 = "Биндеры для бумаги 51 мм";
+        let product3 = "Ручка шариковая синяя";
+
+        let productPrice1 = 280.25;
+        let productPrice2 = 56;
+        let productPrice3 = 12.50;
+
+        let productQuantity1 = prompt(`Укажите количество продуктов '${product1}', цена ${productPrice1}`, 0);
+        let productQuantity2 = prompt(`Укажите количество продуктов '${product2}', цена ${productPrice2}`, 0);
+        let productQuantity3 = prompt(`Укажите количество продуктов '${product3}', цена ${productPrice3}`, 0);
+
+        // Выведите на экран сколько пользователю нужно заплатить за каждый продукт по отдельности и всего за все продукты
+        // используйте форматированный вывод
+        let totalPriceProduct1 = productPrice1 * productQuantity1;
+        let totalPriceProduct2 = productPrice2 * productQuantity2;
+        let totalPriceProduct3 = productPrice3 * productQuantity3;
+
+        let delivery = confirm("Доставку заказывать будете?");
+        let deliveryText = "Цена доставки";
+        let costOfDelivery = 40;
+
+        if (delivery) {
+            let total = totalPriceProduct1 + totalPriceProduct2 + totalPriceProduct3 + costOfDelivery;
```

Comment:
Please think about how you can minify your code. You have to calculate your total and message as usually and only in case when delivery was chosen add delivery cost to the total